src/dawn/Optimizer/PassStageSplitter.cpp

```diff
@@ -28,6 +28,19 @@
 
 namespace dawn {
 
+namespace {
+
+static void reportDataDependencyInsideConditionalBlock(
```

Comment:
so in c++ static and the anonymous namespace do exactly the same thing if used on a function.

the rule of thumb is to keep anonymous namespaces as small as possible since it's really tedious to figure out if functions are inside one on not. use them when you need to declare static classes or structs like here
https://github.com/MeteoSwiss-APN/dawn/blob/0842c266d23eada1a47a9140834e2f1564b12003/src/dawn/IIR/DependencyGraphAccesses.cpp#L273
but use the static keyword on functions if possible (as it is more explicit since it's next to the function and not (potentially) several lines up.

Reply:
Ok, i will only use the static keyword and remove the anonymous namespace.